docs/Validation-and-Serialization.md

```diff
@@ -367,6 +367,8 @@ fastify.setErrorHandler(function (error, request, reply) {
 })
 ```
 
+If you want custom error response in schema without headaches and quickly, you can take a look at [here](https://github.com/epoberezkin/ajv-errors)
```

Comment:
Can you also link Ajv i18n?

Reply:
I linked Ajv i18n [here](https://github.com/fastify/fastify/pull/1588/files#diff-71112d6db09dd080118d76d15c362f4eR503)